build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'java'
+}
+
+group 'org.example'
+version '1.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.2'
+    implementation 'commons-logging:commons-logging:1.2'
+    testImplementation 'org.mockito:mockito-core:4.3.1'
+}
+
```

Comment:
add checkStyle and pmd.

Reply:
used checkstyle and pmd, doing some changes